stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -99,7 +99,7 @@ public class ChatClient internal constructor(
     public val config: ChatClientConfig,
     private val api: ChatApi,
     private val socket: ChatSocket,
-    private val notifications: ChatNotifications,
+    @InternalStreamChatApi public val notifications: ChatNotifications,
```

Comment:
We need to make it public for testing purposes, right? Maybe we can leave it for now but it would be nice to improve this in the future

Reply:
Yep, it is needed because on the tests side we need to override this collaborator and it is not accessible in another case.
Hope AGP having support  for `java-test-fixtures` plugin soon and we can avoid all this ugly changes